fdbserver/VersionedBTree.actor.cpp

```diff
@@ -6442,8 +6385,13 @@ class KeyValueStoreRedwoodUnversioned : public IKeyValueStore {
 
 		wait(cur.seekGTE(key, 0));
 		if (cur.isValid() && cur.get().key == key) {
-			return cur.get().value.get();
+			// Return a Value whose arena is the source page's arena
```

Comment:
Good catch, I wrote this comment when its statement was true, but then I changed it because using the same Arena would mean that any dependencies added to the result later by the caller will likely be added to the Page arena too, as there will usually be space for new ArenaBlock pointers in the Page Arena's current block.